src/app/components/Transactions/RuntimeTransactions.tsx

```diff
@@ -103,7 +103,7 @@ export const RuntimeTransactions: FC<TransactionsProps> = ({
         ...(verbose
           ? [
               {
-                content: <RuntimeTransactionMethod method={transaction.method} truncate />,
+                content: <RuntimeTransactionMethod transaction={transaction} truncate />,
```

Comment:
It looks cool in details, but do we really want it in lists which show more general info ?
![Screenshot from 2024-10-08 18-34-13](https://github.com/user-attachments/assets/4db24e7c-70c2-454d-9218-876e11fe961d)

cc: @donouwens